data/remnant/remnant 1 introduction.txt

```diff
@@ -294,8 +247,8 @@ mission "Remnant: Defense 1"
 	source "Caelian"
 	to offer
 		has "remnant: blood test pure"
+		not "remnant: defense delay"
 		not "Remnant defense delay"
```

Comment:
We don't really need to bother covering this case. If someone updates their game and gets this mission offered again quicker than they otherwise would have then whatever, not a big deal.
```suggestion
```